src/plugins/veevalidate.js

```diff
@@ -4,6 +4,8 @@ import { ValidationProvider, ValidationObserver } from 'vee-validate'
 import en from '../locales/en'
 import es from '../locales/es'
 import cn from '../locales/cn'
+import ua from '../locales/ua'
+import fr from '../locales/fr'
```

Comment:
Are you also contributing French locale?

Reply:
This line fixes validation issue - translation messages does not displaying properly in French

Reply:
But we don’t have French locales in the repo